examples/ios/swift/Migration/MigrationExample.swift

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Foundation
+import RealmSwift
+
+struct MigrationExample {
```

Comment:
Why not have these methods just be in the AppDelegate class? Why make the user have to jump to another file?

Reply:
My idea behind this was a separation of concerns:
The `AppDelegate` manages the app lifecycle by implementing the `AppDelegate` `protocol` functions.
`MigrationExample` manages the actual migration example.

Would you prefer to move this to the `AppDelegate`?

Reply:
Problem is, `MigrationExample` isn't a "thing"– having it as a struct is a completely arbitrary choice for the purpose of the example, and we should avoid anything being "for the purpose of the example" as much as possible.

If you're hesitant to put these in the `AppDelegate` itself, they can just be private global functions in the app delegate file. It's not like they have any state.